### PR TITLE
locing mechanism

### PR DESCRIPTION
**Pull request by TheRyeGuyWhoWillNowDie**
smh someone BETTER get fired for THIS BLUNDER

# Changelog

:cl:  
spellcheck: locing mechanism -> locking mechanism
spellcheck: closly -> closely
spellcheck: outfited -> outfitted
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/ballistic/bow.dm [hunk 1]
```diff
@@ -125,7 +125,7 @@
 	
 /obj/item/gun/ballistic/bow/crossbow
 	name = "wooden crossbow"
-	desc = "A bow with a locing mechanism that more closly resembles a modern gun."
+	desc = "A bow with a locking mechanism that more closly resembles a modern gun."
```
Comment (SomeguyManperson) [inline code/modules/projectiles/guns/ballistic/bow.dm]:
```suggestion
	desc = "A bow with a locking mechanism that more closely resembles a modern gun."
```
smh

Comment (TheRyeGuyWhoWillNowDie) [reply code/modules/projectiles/guns/ballistic/bow.dm]: yo what the fuck

Comment (tattax) [reply code/modules/projectiles/guns/ballistic/bow.dm]: same with the bottom one

File: code/modules/projectiles/guns/ballistic/bow.dm [hunk 2]
```diff
@@ -135,7 +135,7 @@
 	
 /obj/item/gun/ballistic/bow/crossbow/magfed
 	name = "wooden magfed crossbow"
-	desc = "A bow with a locing mechanism that more closly resembles a modern gun. This one seems to be outfited with a automatic loading mechanism."
+	desc = "A bow with a locking mechanism that more closely resembles a modern gun. This one seems to be outfited with a automatic loading mechanism."
```
Comment (SomeguyManperson) [inline code/modules/projectiles/guns/ballistic/bow.dm]:
```suggestion
	desc = "A bow with a locking mechanism that more closely resembles a modern gun. This one seems to be outfitted with a automatic loading mechanism."
```

Comment (TheRyeGuyWhoWillNowDie) [reply code/modules/projectiles/guns/ballistic/bow.dm]: what the shit

File: code/modules/projectiles/guns/ballistic/bow.dm [hunk 3]
```diff
@@ -135,7 +135,7 @@
 	
 /obj/item/gun/ballistic/bow/crossbow/magfed
 	name = "wooden magfed crossbow"
-	desc = "A bow with a locing mechanism that more closly resembles a modern gun. This one seems to be outfited with a automatic loading mechanism."
+	desc = "A bow with a locking mechanism that more closely resembles a modern gun. This one seems to be outfitted with a automatic loading mechanism."
```
Comment (tattax) [inline code/modules/projectiles/guns/ballistic/bow.dm]:
```suggestion
	desc = "A bow with a locking mechanism that more closely resembles a modern gun. This one seems to be outfitted with an automatic loading mechanism."
```

Comment (TheRyeGuyWhoWillNowDie) [reply code/modules/projectiles/guns/ballistic/bow.dm]: it just keeps getting worse

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**